test/system/test_driver.sh

```diff
@@ -0,0 +1,676 @@
+  echo "CAM_TASKS:         Default = (depends on system)"
+  echo "CAM_TESTDIR:       Default = <user_scratch_dir>/test-driver.<jobid>"
+  echo ""
+  echo "Less common ENVIRONMENT variables"
+  echo "CALDERA_BATCHQ:    Default = caldera"
+  echo "CAM_RBOPTIONS:     Default = build_only"
+  echo "CAM_SOFF:          Default = none (stop of first test fail if TRUE)"
+  echo "CIME_MODEL:        Default = none (should be set to cesm)"
+  echo "EMAIL:             Default = $USER@ucar.edu"
+  echo "SUMMARY_FILE:      Default = `pwd -P`/cam_test_summaries}"
+  exit $1
+}
+
+##
+## Error output function (should be handed a string)
+##
+perr() {
+  echo -e "\nERROR: ${@}\n"
+  help 1
+}
+
+## These variables may be overridden from the user's environment
+EMAIL=${EMAIL:-"${USER}@ucar.edu"}
+
+# These variables may be modified by script switches (./test_driver.sh -h)
+cam_email_summary=false
+cesm_test_suite="aux_sima"
+force=false
+gmake_j=0
+interactive=false
+use_existing=''
+namelists_only=false
+batch=false
+
+# Understand where we are and where the CAM root and CIME reside
+if [ -n "${CAM_ROOT}" ]; then
+   # The user specified a CAM_ROOT, make sure it exists and that this
+   # script was called from its test directory.
+   if [ -d "${CAM_ROOT}" ]; then
+     test_dir="${CAM_ROOT}/test/system"
+     if [ -f "${test_dir}/test_driver.sh" ]; then
+       # Check against this script.
+       script_dir="$( cd $( dirname $0 ); pwd -P )"
+       script_file="${script_dir}/test_driver.sh"
+       if [ "${test_dir}/test_driver.sh" != "${script_file}" ]; then
+         perr "CAM_ROOT test dir is ${test_dir} but script is ${script_file}"
+       fi # Else, everything is fine
+     else
+       perr "No test_driver.sh found in ${test_dir}"
+     fi
+   else
+     perr "CAM_ROOT, '${CAM_ROOT}', does not exist"
+   fi
+else
+  # The user did not specify CAM_ROOT, find it relative to this script
+  test_dir="$( dirname $0 )"
+  export CAM_ROOT="$(dirname $(dirname $( cd ${test_dir}; pwd -P )))"
+fi
+# Now, find CIME_ROOT, first try a CAM standalone checkout
+if [ -d "${CAM_ROOT}/cime/scripts" ]; then
+  export CIME_ROOT="${CAM_ROOT}/cime"
+else
+  export CIME_ROOT="$( dirname $( dirname ${CAM_ROOT} ) )/cime"
+  if [ ! -d "${CIME_ROOT}/scripts" ]; then
+    perr "No CIME found from CAM_ROOT = '${CAM_ROOT}'"
+  fi
+fi
+
+# Initialize variables which may not be set
+submit_script_cime=''
+
+while [ "${1:0:1}" == "-" ]; do
+    case $1 in
+
+        --baseline-dir )
+            if [ $# -lt 2 ]; then
+                perr "${1} requires a directory name)"
+            fi
+            baseline_dir="${2}"
+            shift
+            ;;
+
+        --no-baseline ) no_baseline=false
+             ;;
+
+        -b ) export CAM_BASEBACK="YES"
+             ;;
+
+        --cesm )
+            if [ $# -lt 2 ]; then
+                perr "${1} requires a CESM test name or test suite name (e.g., aux_cam_sima)"
```

Comment:
should this be "aux_sima"?

Reply:
Yup - Jesse suggested the name change, and I missed a spot!

Reply:
Github might be being weird for me but I think you missed this change.

Reply:
My bad.  I stumbled across this a week or two ago and fixed it.  I guess it wasn't in a version I committed, and I didn't check (because I remembered doing it!)  It is now fixed!